### ISSUE TITLE
CAVT 72 

### ISSUE THREAD

**Issue by PietroLiuzzo** [BetaMasaheft/Documentation#672]
I cannot find anything matching n. 72 in CAVT. Alius sermo de nativitate Henoch. this is an example search. http://betamasaheft.eu/as.html?query=birth+Enoch&references=&xmlid=&work-types=work

should I create a new record or did I simply miss the correct english title present in the data?

**Comment by PietroLiuzzo** [comment]
I also cannot find a record for CAVT 90 De connnoratione Abrahae et Sarae In Aegypto

**Comment by PietroLiuzzo** [comment]
117 Mors Joseph ?
141 Arde'et et Acta Moysi (aethiopice) ?

**Comment by PietroLiuzzo** [comment]
230 Historia Josias ?

**Comment by MassimoVilla** [comment]
Alius sermo de nativitate Henoch (CAVT no. 72): it seems to have been treated in Siegbert Uhlig, _Das äthiopische Henochbuch_, Gütersloh 1984 (Jüdische Schriften aus hellenistisch-römischer Zeit 5), pp. 466ff. Unfortunately I do not have that book at hand now, if none can check the reference I will do as soon as I am in Hamburg. I suspect a new record is anyway needed.

**Comment by MassimoVilla** [comment]
CAVT 90 De connnoratione Abrahae et Sarae In Aegypto: I have two options

- either a homily by Ephrem of Syria, Dǝrsāna ʾAbrǝhām wa-Sǝra ba-Gǝbṣ (LIT1281Dersan, cfr. EAe II, 137b)
- or a non-independent portion of the Kǝbra Nagaśt recounting Abreham and Sarah's descent into Egypt.

**Comment by MassimoVilla** [comment]
141 Arde'et et Acta Moysi (aethiopice):
there are five records in the app named Maṣḥafa ʾardǝʾǝt (Book of the disciples), depending on the version, either Christian or Betä Ǝsraʾel, and again if long or short recension. The one at issue should be the Betä Ǝsraʾel version (= LIT3945ArdeetBE), in which Moses (not Jesus) reveals the secret names of God to his disciples. But there are more magical texts known as "Acts (or Prayer) of Moses", two at least published, so this identification still requires more study. Is there any clue in the Clavis?

**Comment by MassimoVilla** [comment]
CAVT 72 (Alius sermo de nativitate Henoch) is  the fourth book of the Maṣḥafa mǝśṭira samāy wa-mǝdr (LIT1954Mashaf). Clavis number added to the textpart.

**Comment by PietroLiuzzo** [comment]
could you commit that? what does to the textpart means? thanks!

**Comment by MassimoVilla** [comment]
`  <div type="textpart" subtype="chapter" n="4" xml:id="Treatise4">
               <label>
                  <foreign xml:lang="gez">Dǝrsān</foreign> about the birth of Enoch, <foreign xml:lang="gez">lǝdatu la-Henok</foreign> 
               </label>
               <listBibl type="clavis">
                  <bibl  type="CAVT">
                     <ptr target="bm:CAVT"/>
                     <citedRange unit="item">72</citedRange>
                  </bibl>
               </listBibl>`

**Comment by PietroLiuzzo** [comment]
I see. Thanks! Was this documented in the guidelines? we will have to test the viability of this approch. would not a CAVT id justify a separate record?

**Comment by PietroLiuzzo** [comment]
please check if the current visualization of this information is satisfactory and that it is not deceiving.

**Comment by MassimoVilla** [comment]
In reality the fourth book of the Maṣḥafa mǝśṭira samāy wa-mǝdr is never found as an autonomous text in any witness of the work. I do not think there is evidence for a separate record,

**Comment by PietroLiuzzo** [comment]
good, then we only need to make sure that the way the information is visualized at the moment does not deceive a user to think that the whole work equals CAVT 72, right?

**Comment by MassimoVilla** [comment]
The CAVT reference appears in the app but perhaps it needs more visibility.

**Comment by MassimoVilla** [comment]
no, this is exactly the point. The reference is under Chapter 4, but perhaps one would like to see it also under the main title of the work, but obviously referred only to chapter 4

**Comment by MassimoVilla** [comment]
http://betamasaheft.eu/works/LIT1954Mashaf/text?per-page=1&start=14&keyword=&date-range=&language=

**Comment by PietroLiuzzo** [comment]
that obviously is no piece of cake :) but I will see what can be done. thanks!

**Comment by MassimoVilla** [comment]
thank you!

**Comment by MassimoVilla** [comment]
CAVT 117 (Mors Joseph) = LIT2458Testam (Testament of Joseph, or ዜናሁ፡ ለዮሴፍ፡)

**Comment by MassimoVilla** [comment]
CAVT 230 (Historia Josias) refers to a work probably dealing with the Israelite king and inserted as an apocryphal appendage to the canonical Jeremiah in an unspecified and not investigated manuscript from a church in Tegray. The information was given by Roger Cowley in 1971.
In my view, at present it is too little to create a new record out of this remark.
